src/pipeline_v3/core/index_manager_atomic.py

```
@@ -169,48 +171,50 @@ async def add_document_atomic(
-                    "artifact_data": artifact_data
+                    "artifact_data": artifact_data,
                 },
-                metadata=metadata
+                metadata=metadata,
             )
-            
+
             # Execute atomic transaction
-            success, errors, details = await self.transaction_coordinator.execute_transaction(operation)
-            
+            success, errors, details = await self.transaction_coordinator.execute_transaction(
```

Pylint (3.3.7)

[warning] Unused variable 'details'

(W0612)